joomla/joomla-cms
# Add link to JED - schema plugins category in schema plugin form

Pull Request for PR #42710 .

### Summary of Changes

Since not all schema types can be maintained in the core of Joomla itself, a new category was created in the JED (Joomla Extension Directory) especially for extending functionality with further plugins from the community.

The greatest strength of Joomla! is the community - the publication of additional schema plugins in the JED is a great opportunity for all of us. 

This PR embeds a note and a direct link to this category in JED directly in the form of the schema system plugin, so that all users have easy access to additional schema type plugins from the great developers in the community.

### Testing Instructions

If not already done configure the schema system plugin first.
To avoid misunderstandings, the notice is only displayed after the core function has been activated
![grafik](https://github.com/joomla/joomla-cms/assets/64533137/b2c9baa4-8a9a-4d89-ac7b-e039cfb1e6ad)

After activation, a note with a direct link to the schema category in the JED should be displayed at the end of the form.

### Actual result BEFORE applying this Pull Request

No additional hint and link to JED in schema form.

### Expected result AFTER applying this Pull Request

Note with a direct link to the schema category in JED should be displayed at the end of the form.

![grafik](https://github.com/joomla/joomla-cms/assets/64533137/3d0c5590-c7a5-4889-bb31-2bba411ae675)

![grafik](https://github.com/joomla/joomla-cms/assets/64533137/2832b022-dfc5-4ab3-9aac-abf9aa1e4578)

### Link to documentations
Please select:
